### DISCUSSION TITLE
Factory reset Tokens

### DISCUSSION THREAD

**Discussion by iansampaio** [six7/figma-tokens#1212] [General]
I experienced a few strange behaviors with Tokens in a specific file, for example suddenly assigning hex codes instead of the existing styles, and overall taking long to update document (update progress notification) although my json was not even big compared to another file. 
So I decided to start from zero by... deleting all my tokens (cleared the entire json and kept only `{}` ) and even deleted my set and created a new one, to then reimporting my styles back.
However somehow it stubbornly continues to find the newly added colors and updates the document with them.

Does anyone know a way to "factory reset" Tokens? 

Bonus: The way to remove tokens from elements takes a lot of steps, now imagine doing that to a lot of elements... is there a better way to do this?

**Comment by six7** [comment]
Do you want to reset the tokens JSON or do you want to remove any tokens applied on any layers? If the latter, this can be done by selecting all, then in the plugin going to Inspect and checking Deep inspect. This will show all tokens applied to any layer. You can just select all and remove then.

**Comment by iansampaio** [comment]
Thanks @six7, the latter! It's exactly what I was looking for!
